website/django/data/migrations/0026_report_change_type.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.4 on 2018-02-16 17:31 and edited by zack fischmann
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('data', '0025_auto_20180116_1508'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            """
+            INSERT INTO data_changetype VALUES (7, 'none'); 
```

Comment:
variants are currently only added to the database if they have a changetype value (variants without changetypes are already in the database and therefore don't need to be added again)

unlike variants, reports may be added to the database even if they're identical to existing reports, hence the need for the 'none' changetype

Reply:
Would be nice to integrate this comment in the code.